Alexander_etal_OpEd_Reanalaysis.tex

```diff
@@ -103,16 +103,18 @@ \subsection{Directly linking secondary data products to provenance of work-flow}
 archived again with Zenodo using the same approach as above.
 Moreover, the Zenodo archival system will then automatically indicate
 the presence of other versions of a given repository such that a user
-might be sure to use the newest version of an assembly. In future,
+might be sure to use the newest version of an assembly. In the future,
 such an approach might be further complemented by the integration of a
 JSON Linked Data file detailing the metadata for the assembly product,
 such as the pipeline used and previous versions of the assemblies.
 
+It should be noted that uploading the assemblies to Zenodo was not an automated process. Addition of new versions of files on Zenodo must be manually curated. Since the start of this project, the Open Science Framework (OSF) and the accompanying automated command-line client, osfclient has been established. In the future, large-scale projects such as the assemblies created in this analysis will benefit from the integration of osfclient by automatically uploading data products to an OSF projects, which generate an OSF-specific DOI.
```

Comment:
@ljcohen how would you feel about this being a footnote? I think it needs to be in there but stands out a bit on its own.

Reply:
Yes, a footnote is fine! Wanted to make sure the manual uploading aspect of Zenodo is mentioned somewhere.

Reply:
Cool! I am going pull your comments over into my latest version (as it is a bit ahead and I forsee merger difficulty). I will tag you when I push.